Sparkle/SPUUpdater.m

```diff
@@ -216,8 +217,8 @@ - (BOOL)checkIfConfiguredProperly:(NSError * __autoreleasing *)error
         }
     }
     
-    BOOL hasPublicDSAKey = [self.host publicDSAKey] != nil;
-    if (!hasPublicDSAKey) {
+    BOOL hasPublicKey = self.host.publicKeys.dsaPubKey != nil || self.host.publicKeys.ed25519PubKey != nil;
+    if (!hasPublicKey) {
         // If we failed to retrieve a DSA key but the bundle specifies a path to one, we should consider this a configuration failure
```

Comment:
This method `-checkIfConfiguredProperly` has a couple `DSA` -> `EdDSA` log strings that should be updated (the `publicDSAKeyFileKey` case is ok-ish since edDSA doesn't have equivalent filename option).